openpmd_viewer/openpmd_timeseries/data_reader/data_reader.py

```diff
@@ -55,6 +64,28 @@ def list_iterations(self, path_to_dir):
                 h5py_reader.list_files( path_to_dir )
             # Store dictionary of correspondence between iteration and file
             self.iteration_to_file = iteration_to_file
+
```

Comment:
For `h5py`, let us raise a symmetric error to the one below:
```suggestion
            if len(iterations) == 0:
                raise RuntimeError(
                    "Found no valid files in directory {0}.\n"
                    "Please check that this is the path to the openPMD files."
                    "(valid files must have the extension '.h5')"
                    .format(path_to_dir))
```